src/main/adapters/home.js

```diff
@@ -32,6 +32,13 @@ exports.HomeAdapter = class {
       .asCallback(callback)
   }
 
+  getModulesDisplayNames (callback) {
+    return this.server.modulesPromise
+      .then(modules => modules.map(({ name, displayName }) => ({ [name]: displayName || name }))
```

Comment:
Bluebird promises have thier own version of map and reduce it is better to use it directly see http://bluebirdjs.com/docs/api/map.html